### PR TITLE
Added extension marker to enumerations

### PR DESCRIPTION
**Pull request by FabianEckermann**
Added extension marker to enumerations as defined in the standard (X.691)

### CODE REVIEW

**Comment by eerimoq** [comment]
Please add unit tests for the new functionality, and I'll review the PR and then merge.

File: asn1tools/codecs/uper.py [hunk 1]
```diff
@@ -1044,6 +1058,14 @@ def compile_type(self, name, type_descriptor, module_name):
 
         return compiled
 
+    def get_extension(self, values):
+        extension = None
+        
+        for value in values:
+            if value == '...':
+                extension = []
+        return extension
```
Comment (eerimoq) [inline asn1tools/codecs/uper.py]: Does this function return anything but `None`? I mean, the loop iterates over the keys, which are integers. Comparing an integer to `'...'` will never be true. Or am I missing something?

Comment (FabianEckermann) [reply asn1tools/codecs/uper.py]: You are right, I fixed it

**Comment by FabianEckermann** [comment]
I fixed the detection of extension markers for enumerations and updated the tests. However test_uper.py fails as the hardcoded encoded_message does not fit the encoded one as there are extension markers within test_lpp_14_3_0 that were ignored before

**Comment by eerimoq** [comment]
Ok, update `test_lpp_14_3_0` so it passes with your changes. Also, add encode/decode testing in `test_encode_all_types` and `test_decode_all_types` of `Enumerated2` with value `two`.

**Comment by FabianEckermann** [comment]
This should be a valid way to get the max key of the root element. I also added the tests you proposed.

**Pull request merged**